pkg/plugin/trans.go

```diff
@@ -214,6 +214,33 @@ func (r resourceTransition[T]) handleDeleted(currentlyMigrating bool) (verdict s
 	return verdict
 }
 
+// handleAutoscalingDisabled updates r.node with changes to clear any buffer and capacityPressure
+// from r.pod
+//
+// A pretty-formatted summary of the changes is returned as the verdict, for logging.
+func (r resourceTransition[T]) handleAutoscalingDisabled() (verdict string) {
```

Comment:
I don't think there's anything gained by declaring `verdict` here

Reply:
Leaving as-is, to match the rest of the file. May change in a follow-up PR, but undecided.